relay-server/src/metrics_extraction/spans/mod.rs

```diff
@@ -1,21 +1,22 @@
 use crate::metrics_extraction::spans::types::{SpanMetric, SpanTagKey};
 use crate::metrics_extraction::transactions::types::ExtractMetricsError;
 use crate::metrics_extraction::utils::extract_http_status_code;
-use crate::metrics_extraction::utils::http_status_code_from_span;
 use crate::metrics_extraction::utils::{
     extract_transaction_op, get_eventuser_tag, get_trace_context,
 };
 use crate::metrics_extraction::IntoMetric;
-use itertools::Itertools;
 use once_cell::sync::Lazy;
 use regex::Regex;
 use relay_common::{EventType, UnixTimestamp};
-use relay_filter::csp::SchemeDomainPort;
 use relay_general::protocol::Event;
 use relay_general::types::{Annotated, Value};
 use relay_metrics::{AggregatorConfig, Metric};
 use std::collections::BTreeMap;
 
+mod cache_tags;
+mod common_tags;
+mod db_tags;
+mod http_tags;
```

Comment:
I don't really like these module names, but I can't think of better suggestions.

Reply:
I do not really see the reason to do it in first place. 
In my opinion the good separation would be to extract all tags-related code into `tags.rs` module and that's it. Keep it simple, also all code concerning tags is together.

Reply:
If I understand correctly, your suggestion is to move these into the same file?

Reply:
Exactly! And we might have to rename some of the functions to make it more obvious what's extracted.